tests/queries/0_stateless/02377_optimize_sorting_by_input_stream_properties_explain.reference

```diff
@@ -88,6 +90,7 @@ Sorting (Chunk): a ASC
 -- check that correct sorting info is provided in case of only prefix of sorting key is in ORDER BY clause but all sorting key columns returned by query
 -- QUERY: set optimize_read_in_order=1;EXPLAIN PLAN sorting=1 SELECT a, b FROM optimize_sorting ORDER BY a
 Sorting (Global): a ASC
+Sorting (Sorting for ORDER BY)
 Sorting (Global): a ASC
-Sorting (Stream): a ASC
+Sorting (Chunk): a ASC, b ASC
```

Comment:
This is an incorrect property, but it won't be used cause plan is already built.
We should probably fix it later.